dataset_config/1000_genomes/ui.json

```diff
@@ -8,6 +8,7 @@
 //   listed here. ui_facet_description is optional.
 {
   "search_placeholder_text": "Try typing exome or center",
+  "enable_search_values": true,
```

Comment:
I'm thinking of keeping this flag even after feature is done. Please make these changes in both this file and template/ui.json.

- Move flag to bottom of file.
- Description:
// enable_search_values:
//   Whether to search on facet values.
//   If this is true, search box will search BigQuery column contents in addition to
//   BigQuery column name and description. Search results will take time to appear.
//   If this is false, search box will search BigQuery column name and description.
//   Search results will appear instantly.
- Set to true in both files

Reply:
Also, for both `ui.json`:

// search_placeholder_text:
//   Optional. What to show in the search box by default. Adjust the terms for
//   your dataset. We recommend:
//   - A word that appears in BigQuery column descriptions, eg "center" for 1000 Genomes
//   - A prefix that appears in BigQuery column contents, eg "pat" for 1000 Genomes

Reply:
Sorry, one more change. (It didn't seem worth me making a separate PR for this.) In both files:

"search_placeholder_text": "Search center or pat",